config/default.yml

```diff
@@ -262,3 +262,14 @@ CommitMsg:
 
   TrailingPeriod:
     description: 'Checking for trailing periods in subject'
+
+PostCommit:
+  ALL:
+    requires_files: false
+    required: false
+    quiet: false
+
+  GitGuilt:
```

Comment:
@lencioni That's largely how I use it. It's good to know whose work you're clobbering. But I agree, this shouldn't be a default.